ibu-imager/internal/seedcreator/seedcreator.go

```diff
@@ -356,3 +432,35 @@ func (s *SeedCreator) backupOstreeOrigin(statusRpmOstree *ostree.Status) error {
 	log.Println("Backup of .origin created successfully.")
 	return nil
 }
+
+// getEtcdImageFromStaticDefinition reads the static definition of the etcd pod
+// and returns the image used by this pod.
+func getEtcdImageFromStaticDefinition() (string, error) {
+
+	// Read the YAML file
+	yamlData, err := os.ReadFile(etcdStaticPodFile)
```

Comment:
Maybe we can read file into pod struct?

Reply:
Updated `GetEtcdImageFromStaticDefinition` function in [7ee04ba](https://github.com/openshift-kni/lifecycle-agent/pull/35/commits/7ee04ba680d0c631239e2545efb79221fea30058).